src/EnergyPlus/api/datatransfer.cc

```diff
@@ -126,9 +126,9 @@ int apiDataFullyReady(EnergyPlusState state)
 {
     auto *thisState = reinterpret_cast<EnergyPlus::EnergyPlusData *>(state);
     if (thisState->dataPluginManager->fullyReady) {
-        return 0;
+        return 1;
     }
-    return 1;
+    return 0;
```

Comment:
#9998

Reply:
Yeah.  I do agree this is more meaningful.  In my mind, 0 means success and 1 means failure, which translated to 0 being true and 1 being false.   I understand the confusion and I think I support this change.  But.  This will be a breaking change for any user programs which assume the other.  A correction, yes, but still a breaking change.